### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by rorticus**
README Updates!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,9 @@
 
 This package provides functional shims for ECMAScript, access to the Typescript helpers, and a quick way to include the polyfills needed to run Dojo 2 in the browser.
 
-It is targeted at providing function shims for ECMAScript 6 and beyond targeted at ECMAScript 5.  It is different
-other solutions of shimming or polyfilling functionality, in that it does not provide the functionality via
-augmenting the built in classes in the global namespace.
+It is targeted at providing function shims for ECMAScript 6 and beyond targeted at ECMAScript 5.  It is different other solutions of shimming or polyfilling functionality, in that it does not provide the functionality via augmenting the built in classes in the global namespace.
```
Comment (dylans) [inline README.md]: Not your typo, but "different other" should be "different than other"

File: docs/has.md [hunk 2]
```diff
@@ -2,8 +2,7 @@
 
 ## Detecting Features
 
-The default export of `dojo-core/has` is a function which accepts a single parameter: the name of the feature to test for.
-If the feature is available, a truthy value is returned, otherwise a falsy value is returned:
+The default export of `dojo-core/has` is a function which accepts a single parameter: the name of the feature to test for.If the feature is available, a truthy value is returned, otherwise a falsy value is returned:
```
Comment (dylans) [inline docs/has.md]: for.If -> for. If

File: docs/has.md [hunk 3]
```diff
@@ -35,6 +31,4 @@ hasAdd('typedarray', function () {
 
 ## Accessing the Feature Cache
 
-`dojo-core/has` maintains object hashes containing keys that correspond to all features that have been both
-registered _and_ requested. The value associated with each feature name key corresponds to that feature's availability
-in the current environment. The object hash containing evaluated features is accessible via the `cache` export.
+`dojo-core/has` maintains object hashes containing keys that correspond to all features that have been bothregistered _and_ requested. The value associated with each feature name key corresponds to that feature's availability in the current environment. The object hash containing evaluated features is accessible via the `cache` export.
```
Comment (dylans) [inline docs/has.md]: bothregistered -> both registered

**Review by dylans — CHANGES_REQUESTED** [review]
a few typos

File: docs/has.md (hunk 2 above)
Comment (rorticus) [reply docs/has.md]: So I actually just deleted this... it is not linked to and is a duplicate of the @dojo/core/has doc

File: docs/has.md (hunk 3 above)
Comment (rorticus) [reply docs/has.md]: same as above

File: README.md (hunk 1 above)
Comment (rorticus) [reply README.md]: Fixed!

**Review by dylans — APPROVED** [review]

**Pull request merged**